cmd/root.go

```diff
@@ -76,6 +84,16 @@ This allows users to schedule kubernetes workloads on nodes that aren't running
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx, cancel := context.WithCancel(context.Background())
 
+		sig := make(chan os.Signal, 1)
+		signal.Notify(sig, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
IIRC this doesn't work on Windows. Should use `os.Interrupt` instead. See https://stackoverflow.com/a/35683558

Reply:
I just realized I may have assumed wrongly that it's intended for the VK to also run on Windows.

Reply:
Yep, will fix.